docker-compose/monitor/jaeger-v2-config.yml

```diff
@@ -0,0 +1,43 @@
+service:
+  extensions: [jaeger_storage, jaeger_query]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [jaeger_storage_exporter, spanmetrics]
+    metrics/spanmetrics:
+      receivers: [spanmetrics] 
+      exporters: [prometheus] 
+
+
+extensions:
+  jaeger_query:
+    trace_storage: some_storage
+    metric_storage: some_metrics_storage
+  jaeger_storage:
+    backends:
+      some_storage:
+        memory:
+          max_traces: 100000
+    metric_backends:
+      some_metrics_storage:
+        prometheus:
```

Comment:
Don't you need to pass the address? The default value in the config might be localhost:9090, but in `docker compose` context it would need to be `prometheus:9090`, which is specific to the actual environment and therefore must be explicitly specified.

We should really use the env variable substitution in this config for the Prom host, and define env variable in the compose file (to reduce the coupling)

Reply:
@yurishkuro I am not sure how the config file can read the env variable. I can define a env variable in the jaeger service which holds the ip of the prometheus container but not sure how to pass it to the yaml file.

Reply:
https://opentelemetry.io/docs/collector/configuration/#environment-variables

Reply:
Understood. I guess we have to go ahead without having a default

Reply:
But since I am using docker compose to set the env variable. We can set the default value there.

Reply:
still need to pass the address, logs show that it's trying to query `localhost` which wouldn't work
```
  jaeger-1      | 2024-07-20T20:49:32.015Z	info	metricsstore/reader.go:106	Prometheus reader initialized	{"kind": "extension", "name": "jaeger_storage", "addr": "http://localhost:9090"}
```